src/app/modules/timesheet/services/timesheet.service.ts

```diff
@@ -93,7 +93,7 @@ export class TimesheetService {
     return totalFlatFee;
   }
 
-  public getTimesheetsLocal() {
+  public getLocalstorageTimesheetsList() {
     let timesheetArray = [];
 
     Object.keys(localStorage).forEach(function(name) {
```

Comment:
`name` is not understandable for a `storedItem`,  it's a timesheet, a CRA, something else ?

Reply:
Every `split` under are not clear, we don't understand on what you sort.
I know you don't touch this code on your PR but the continuous refactoring is important to keep a clear code :D